controller/src/beerocks/master/son_management.cpp

```diff
@@ -1720,6 +1720,43 @@ void son_management::handle_bml_message(
         break;
     }
 #endif //BEEROCKS_RDKB
+
+    case beerocks_message::ACTION_BML_TRIGGER_CHANNEL_SELECTION_REQUEST: {
+
+        auto bml_request =
+            cmdu_rx.addClass<beerocks_message::cACTION_BML_TRIGGER_CHANNEL_SELECTION_REQUEST>();
+
+        if (bml_request == nullptr) {
+            LOG(ERROR) << "addClass cACTION_BML_TRIGGER_CHANNEL_SELECTION_REQUEST failed";
+            return;
+        }
+
+        std::string al_mac   = network_utils::mac_to_string(bml_request->al_mac());
+        std::string ruid_mac = network_utils::mac_to_string(bml_request->ruid());
+        std::string key      = database.get_ruid_key_from_parent_mac_and_ruid(al_mac, ruid_mac);
+        Socket *son_slave_sd = database.get_node_socket(key);
+
+        if (son_slave_sd == nullptr) {
+            LOG(INFO) << "Could not find socket for al_mac=" << al_mac << ", ruid=" << ruid_mac;
```

Comment:
This is an error. How does the BML know that the operation failed?

Reply:
I didn't implement response mechanism. Does response mechanism is required?
I use INFO instead of ERROR because I assumed that the user can send invalid data, and it it just will be ignored.

Reply:
So at least change to ERROR, the user should know that he didn't provide the correct parameters, so at least he will see that in the logs.